packages/create-next-app/create-app.ts

```diff
@@ -256,7 +256,7 @@ export async function createApp({
       rename: (name) => {
         switch (name) {
           case 'gitignore':
-          case 'eslintrc': {
+          case 'eslintrc.json': {
```

Comment:
Do we need to keep `eslintrc` for backwards compatibility?

Reply:
The file doesn't get read from anywhere and acts like any regular `eslintrc` file so the previous format doesn't need to be maintained for compatibility. This just updates it to create a `.eslintrc.json` file when a new app is created with `create-next-app` instead of `.eslintrc` per #26857